packages/0x.js/src/types.ts

```diff
@@ -391,14 +391,18 @@ export interface JSONRPCPayload {
     method: string;
 }
 
+// tslint:disable:max-line-length
 /*
- * orderExpirationCheckingIntervalMs: How often to check for expired orders
- * eventPollingIntervalMs: How often to poll the Ethereum node for new events
+ * orderExpirationCheckingIntervalMs: How often to check for expired orders. Default: 50
+ * eventPollingIntervalMs: How often to poll the Ethereum node for new events. Defaults: 200
+ * expirationMarginMs: Amount of time before order expiry that you'd like to be notified of an orders expiration. Defaults: 0
```

Comment:
Let's just make this comment two lines and remove the `tslint:disable:max-line-length`

Reply:
```
* expirationMarginMs: Amount of time before order expiry that you'd like to be notified 
*                     of an orders expiration. Defaults: 0
`